src/CodeGen_ARM.cpp

```diff
@@ -885,6 +883,9 @@ void CodeGen_ARM::visit(const Load *op) {
             return;
         }
 
+#if LLVM_VERSION < 37
```

Comment:
Officially, yes, but IIRC, the PNaCl-LLVM branch is somewhere between 36 and 37, so definitely want to test against that before nuking